code/modules/clothing/gloves/color.dm

```diff
@@ -12,6 +12,7 @@
 	custom_price = PAYCHECK_MEDIUM * 10
 	custom_premium_price = PAYCHECK_COMMAND * 6
 	cut_type = /obj/item/clothing/gloves/cut
+	clothing_traits = list(TRAIT_CHUNKYFINGERS)
```

Comment:
might wanna mention this in their description.

are there any insulated gloves subtypes that might be affected by this change that probably shouldn't be?

Reply:
changed description, there are no subtypes of this gloves that can be affected